### PR TITLE
Secure entry for cleos wallet import

### PR DESCRIPTION
**Pull request by andriantolie**
- Add secure entry for `cleos wallet import` if `--private-key` option is not used
- Add some additional error message to make it clear when key already exists in the wallet

### CODE REVIEW

File: programs/cleos/main.cpp [hunk 1]
```diff
@@ -2204,13 +2204,20 @@ int main( int argc, char** argv ) {
    string wallet_key_str;
    auto importWallet = wallet->add_subcommand("import", localized("Import private key into wallet"), false);
    importWallet->add_option("-n,--name", wallet_name, localized("The name of the wallet to import key into"));
-   importWallet->add_option("key", wallet_key_str, localized("Private key in WIF format to import"))->required();
+   importWallet->add_option("--private-key", wallet_key_str, localized("Private key in WIF format to import"));
    importWallet->set_callback([&wallet_name, &wallet_key_str] {
+      if( wallet_key_str.size() == 0 ) {
+         std::cout << localized("password: ");
```
Comment (heifner) [inline programs/cleos/main.cpp]: Should prompt for "private key" not "password".

Comment (heifner) [reply programs/cleos/main.cpp]: Also the python test scripts need to be updated to use the --private-key option.

Comment (andriantolie) [reply programs/cleos/main.cpp]: Thanks for catching that. PR updated.

**Comment by heifner** [comment]
@wanderingbort Should we add this to 1.0.8?

**Pull request merged**